docs/user.md

```diff
@@ -0,0 +1,235 @@
+| Control Plane | 3     | 4   | 16       | 100          |
+| Compute       | 3     | 4   | 16       | 100          |
+
+
+*Note: These requirements are higher than the [minimum requirements](https://docs.openshift.com/container-platform/latest/installing/installing_bare_metal/installing-bare-metal.html#installation-minimum-resource-requirements_installing-bare-metal) in OpenShift product documentation due to the resource demand of the certification tests.*
+
+### Dedicated Node for Test Environment
+
+If your compute nodes are at or below minimum requirements, it is recommended to run the certification environment on one dedicated node to avoid disruption of the test scheduler. Otherwise the concurrency between resources scheduled on the cluster, e2e-test manager (aka openshift-tests-plugin), and other stacks like monitoring can disrupt the test environment, leading to unexpected results, like eviction of plugins or certification server (sonobuoy pod).
+
+See the troubleshooting section on ways to identify you might need to use a dedicated node test environment below.
+
+The dedicated node environment cluster size can be adjusted to match the table below. Note the differences in the `Dedicated Test` machine:
+
+| Machine       | Count | CPU | RAM (GB) | Storage (GB) |
+| ------------- | ----- | --- | -------- | ------------ |
+| Bootstrap     | 1     | 4   | 16       | 100          |
+| Control Plane | 3     | 4   | 16       | 100          |
+| Compute       | 3     | 4   | 8        | 100          |
+| Dedicated Test| 1     | 4   | 8        | 100          |
+
+#### Environment Setup
+
+1. Choose one node with at least 8GiB of RAM and 4 vCPU
+2. Label node with `node-role.kubernetes.io/tests=""` (certification related pods will schedule to dedicated node)
+3. Taint node with `node-role.kubernetes.io/tests="":NoSchedule` (prevent other pods from running on dedicated node)
+
+*NOTE: certification pods will automatically have node-selectors and taint tolerations if you use the `--dedicated` flag.*
+
+There are two options to accomplish this type of setup:
+
+##### Option A: Command Line 
+
+```shell
+oc label node <node_name> node-role.kubernetes.io/tests=""
+oc taint node <node_name> node-role.kubernetes.io/tests="":NoSchedule
+```
+
+##### Option B: Machine Set 
+
+If you have support for OpenShift's Machine API then you can create a new `MachineSet` to configure the labels and taints. See [OpenShift documentation](https://docs.openshift.com/container-platform/latest/machine_management/creating-infrastructure-machinesets.html#binding-infra-node-workloads-using-taints-tolerations_creating-infrastructure-machinesets) on how to configure a new `MachineSet`. Note that at time of certification testing, OpenShift's product documentation may not mention your infrastructure provider yet! 
+
+Here is a `MachineSet` YAML snippet on how to configure the label and taint as well:
+
+```yaml
+      metadata:
+        labels:
+          openshift-tests: "true"
```

Comment:
It's optional, we could remove this as we are not using it on the plugins and CLI (could you please confirm it?)

Reply:
You are correct, this one label isn't needed.